hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -111,17 +111,21 @@ static uint16_t vm_apicid2vcpu_id(struct acrn_vm *vm, uint8_t lapicid)
 {
 	uint16_t i;
 	struct acrn_vcpu *vcpu;
+	uint16_t cpuid = 0xFFFFU;
```

Comment:
You didn't address my comments in mail

Reply:
Strange,  You can take a look at the source code.   I did change it .
Line 111:
https://github.com/guoch2/acrn-hypervisor/blob/misra/hypervisor/arch/x86/guest/vlapic.c.